custom/icds/templates/icds/manage_ccz_hosting_links.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'hqwebapp/two_column.html' %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% requirejs_main "icds/js/manage_ccz_hosting_links" %}
+
+{% block page_content %}
+{% initial_page_data 'links' links %}
+{% registerurl 'edit_ccz_hosting_link' domain '---' %}
+{% registerurl 'ccz_hosting' domain '---' %}
+
+<div class="container">
+    <div class="row">
+        <div class="page-header">
+            <h1>{% trans "Manage CCZ Hosting Links" %}</h1>
+        </div>
+    </div>
+    <div class="row">
+        {% if links %}
+        <div class="col-sm-4" id="links">
+            <h3>Links</h3>
+            <ul data-bind="foreach: links">
+                <li>
+                    <a data-bind="attr: { href: editUrl}, text: identifier"></a>
+                    <a data-bind="attr: { href: pageUrl}" target="_blank">
+                        <span class="fa fa-arrow-right"></span>
```

Comment:
I don't think it's clear that clicking the name means edit and then the icon is a separate link that goes to the page. I'd do the names in plain text and then display two buttons or links afterwards, one for "edit" and one for "view"/"open". The scheduled reports UI does something like this.

For icons, there's a `fa-external-link` that more commonly means "this will open in a new tab" if you don't like the eye icon that's used for the view button in scheduled reports.

Reply:
added `fa-external-link` but could not add button for edit in a way that makes it look nicer.

Reply:
Okay. Given how few users this page will have, I'm fine with calling that out of scope and revisiting if people find it hard to use.